2022-06-28 16:41:52,958 - werkzeug - _log():224]INFO in _internal:  * Running on http://127.0.0.1:5000 (Press CTRL+C to quit)
2022-06-28 16:41:52,959 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:41:56,200 - app - generate_auth_token():40]ERROR in token: Expected a string value
2022-06-28 16:41:56,200 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:41:56,201 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:41:56] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:43:54,174 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:43:54,321 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:44:00,263 - app - generate_auth_token():40]ERROR in token: Expected a string value
2022-06-28 16:44:00,263 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:44:00,264 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:44:00] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:44:13,266 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:44:13,426 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:44:26,322 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:44:26,507 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:44:29,957 - app - generate_auth_token():40]ERROR in token: Expected a string value
2022-06-28 16:44:29,958 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:44:29,959 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:44:29] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:45:25,165 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:45:25,374 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:45:26,791 - app - generate_auth_token():40]ERROR in token: Expected a string value
2022-06-28 16:45:26,791 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:45:26,792 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:45:26] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:45:58,525 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:45:58,699 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:46:00,343 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:46:00] "[35m[1mPOST /auth/login HTTP/1.1[0m" 202 -
2022-06-28 16:47:02,575 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:47:02,745 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:47:35,097 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:47:35,411 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:47:36,554 - app - generate_auth_token():42]ERROR in token: It is required that you pass in a value for the "algorithms" argument when calling decode().
2022-06-28 16:47:36,554 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:47:36,555 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:47:36] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:47:57,406 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:47:57,585 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:47:58,875 - app - generate_auth_token():42]ERROR in token: It is required that you pass in a value for the "algorithms" argument when calling decode().
2022-06-28 16:47:58,875 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:47:58,877 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:47:58] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:48:23,752 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:48:23,954 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:48:25,138 - app - generate_auth_token():42]ERROR in token: It is required that you pass in a value for the "algorithms" argument when calling decode().
2022-06-28 16:48:25,138 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:48:25,139 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:48:25] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:49:40,431 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:49:40,651 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:49:41,917 - app - generate_auth_token():42]ERROR in token: It is required that you pass in a value for the "algorithms" argument when calling decode().
2022-06-28 16:49:41,917 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:49:41,918 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:49:41] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:51:34,992 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:51:35,196 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:51:45,902 - app - generate_auth_token():42]ERROR in token: It is required that you pass in a value for the "algorithms" argument when calling decode().
2022-06-28 16:51:45,903 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:51:45,908 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:51:45] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:52:37,452 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:52:37,654 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:52:57,700 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:52:57,942 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:52:59,218 - app - generate_auth_token():42]ERROR in token: Algorithm 'RS256' could not be found. Do you have cryptography installed?
2022-06-28 16:52:59,218 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:52:59,219 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:52:59] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:53:36,551 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:53:36,725 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:53:37,799 - app - generate_auth_token():42]ERROR in token: Algorithm 'RS256' could not be found. Do you have cryptography installed?
2022-06-28 16:53:37,799 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:53:37,800 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:53:37] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-28 16:54:12,293 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:54:12,486 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:54:13,762 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:54:13] "[35m[1mPOST /auth/login HTTP/1.1[0m" 202 -
2022-06-28 16:54:37,808 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:54:38,000 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:54:39,448 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:54:39] "[35m[1mPOST /auth/login HTTP/1.1[0m" 202 -
2022-06-28 16:54:48,705 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:54:48,956 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:54:50,258 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:54:50] "[35m[1mPOST /auth/login HTTP/1.1[0m" 202 -
2022-06-28 16:55:36,690 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-28 16:55:36,919 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:55:38,197 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:55:38] "[35m[1mPOST /auth/login HTTP/1.1[0m" 202 -
2022-06-28 16:56:51,122 - werkzeug - _log():224]INFO in _internal:  * Running on http://127.0.0.1:5000 (Press CTRL+C to quit)
2022-06-28 16:56:51,123 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-28 16:56:54,234 - app - generate_auth_token():42]ERROR in token: Signature has expired
2022-06-28 16:56:54,234 - app - post():139]ERROR in user: Object of type type is not JSON serializable
2022-06-28 16:56:54,235 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [28/Jun/2022 16:56:54] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2022-06-29 12:27:41,843 - werkzeug - _log():224]INFO in _internal:  * Running on http://127.0.0.1:5000 (Press CTRL+C to quit)
2022-06-29 12:27:41,849 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 12:29:06,075 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/config.py', reloading
2022-06-29 12:29:06,250 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 12:30:44,777 - app - generate_confirmation_token():12]ERROR in token: 'NoneType' object is not iterable
2022-06-29 12:30:49,713 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 12:30:49] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 12:34:11,848 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 12:34:12,046 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 12:35:41,131 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 12:35:41] "[31m[1mPOST /auth/signup HTTP/1.1[0m" 403 -
2022-06-29 12:36:37,941 - app - generate_confirmation_token():12]ERROR in token: 'NoneType' object is not iterable
2022-06-29 12:36:48,118 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 12:36:48] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 12:44:22,254 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 12:44:22,454 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 12:44:23,614 - app - generate_confirmation_token():12]ERROR in token: 'NoneType' object is not iterable
2022-06-29 12:44:28,491 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 12:44:28] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 12:45:21,318 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-29 12:45:21,559 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 12:46:59,112 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 12:47:03,636 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 12:47:03] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 13:53:00,003 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 13:53:00,023 - app - post():87]ERROR in user: 'print' is undefined
2022-06-29 13:53:00,028 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 13:53:00] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 500 -
2022-06-29 13:55:14,848 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 13:55:15,055 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 13:55:19,578 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 13:55:19,579 - app - post():88]ERROR in user: Could not build url for endpoint 'static'. Did you forget to specify values ['filename']?
2022-06-29 13:55:19,582 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 13:55:19] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 500 -
2022-06-29 13:55:41,824 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 13:55:42,035 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:02:23,016 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:02:28,990 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:02:28] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 14:03:34,316 - app - get():166]ERROR in user: 'NoneType' object is not iterable
2022-06-29 14:03:34,317 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:03:34] "[31m[1mGET /confirm/%3Cclass%20'Exception'%3E HTTP/1.1[0m" 401 -
2022-06-29 14:06:23,278 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:06:23,519 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:06:33,518 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:06:38,703 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:06:38] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 14:07:19,338 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:07:19,613 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:09:07,623 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:09:07,980 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:12:47,849 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 14:12:48,036 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:13:32,738 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 14:13:32,907 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:13:35,457 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:13:35,457 - app - post():88]ERROR in user: Could not build url for endpoint 'static'. Did you forget to specify values ['filename']?
2022-06-29 14:13:35,459 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:13:35] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 500 -
2022-06-29 14:15:12,586 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:15:12,800 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:15:16,435 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:15:16,436 - app - post():88]ERROR in user: Could not build url for endpoint 'static'. Did you forget to specify values ['filename']?
2022-06-29 14:15:16,438 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:15:16] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 500 -
2022-06-29 14:16:07,290 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 14:16:07,479 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:16:11,458 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:16:22,585 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:16:22] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 14:17:10,486 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:17:10,747 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:17:26,805 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:17:35,842 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:17:35] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 14:18:30,265 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:18:30,459 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:18:31,408 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:18:39,431 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:18:39] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 14:19:13,295 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:19:13,472 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:19:17,035 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:19:17] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 500 -
2022-06-29 14:19:21,850 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:19:21] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 500 -
2022-06-29 14:19:37,743 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:19:38,026 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:19:41,750 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:19:41] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 500 -
2022-06-29 14:19:59,242 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:19:59,470 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:20:05,082 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:20:11,697 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:20:11] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 14:20:50,184 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:20:50,462 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:21:20,773 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:21:21,006 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:21:25,996 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:21:25] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 500 -
2022-06-29 14:21:44,057 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:21:44,260 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:21:47,011 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:21:53,264 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:21:53] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 14:21:59,406 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:21:59,569 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:24:41,761 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 14:24:41,945 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:24:52,820 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 14:24:53,027 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:25:20,061 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/__init__.py', reloading
2022-06-29 14:25:20,287 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:25:45,458 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:25:45] "[35m[1mGET / HTTP/1.1[0m" 500 -
2022-06-29 14:25:45,786 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:25:45] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2022-06-29 14:25:45,788 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:25:45] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2022-06-29 14:25:45,887 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:25:45] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2022-06-29 14:26:24,135 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 14:26:24,323 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:26:28,110 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:26:28] "GET / HTTP/1.1" 200 -
2022-06-29 14:28:12,461 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 14:28:12,733 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:28:13,802 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:28:13] "GET / HTTP/1.1" 200 -
2022-06-29 14:29:47,513 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:29:47] "GET / HTTP/1.1" 200 -
2022-06-29 14:30:00,787 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 14:30:00,943 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:30:01,981 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:30:01] "GET / HTTP/1.1" 200 -
2022-06-29 14:33:33,085 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 14:33:33,357 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 14:33:35,393 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:33:35] "GET / HTTP/1.1" 200 -
2022-06-29 14:33:35,591 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:33:35] "GET /static/css/style.css HTTP/1.1" 200 -
2022-06-29 14:33:35,606 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:33:35] "GET /static/images/h1.gif HTTP/1.1" 200 -
2022-06-29 14:33:35,627 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:33:35] "GET /static/images/twitter.svg HTTP/1.1" 200 -
2022-06-29 14:33:35,647 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:33:35] "GET /static/images/instagram.svg HTTP/1.1" 200 -
2022-06-29 14:33:35,632 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:33:35] "GET /static/images/facebook.svg HTTP/1.1" 200 -
2022-06-29 14:33:35,652 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:33:35] "GET /static/images/github.svg HTTP/1.1" 200 -
2022-06-29 14:33:35,743 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:33:35] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2022-06-29 14:37:18,227 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:37:23,393 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:37:23] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 14:38:21,837 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 14:38:26,980 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 14:38:26] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 15:17:14,523 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-29 15:17:14,775 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:19:17,906 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-29 15:19:18,099 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:19:23,676 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 15:19:29,687 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:19:29] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 15:19:35,609 - app - get():165]ERROR in user: 'NoneType' object is not iterable
2022-06-29 15:19:35,610 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:19:35] "[31m[1mGET /confirm/%3Cclass%20'Exception'%3E HTTP/1.1[0m" 401 -
2022-06-29 15:20:36,688 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-29 15:20:36,930 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:23:26,542 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-29 15:23:26,793 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:23:31,947 - app - generate_confirmation_token():13]ERROR in token: 'NoneType' object is not iterable
2022-06-29 15:23:37,154 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:23:37] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 15:24:14,523 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-29 15:24:14,729 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:25:11,578 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 15:25:11,750 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:28:33,964 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 15:28:34,192 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:28:35,382 - app - generate_confirmation_token():12]ERROR in token: 'NoneType' object is not iterable
2022-06-29 15:28:45,316 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:28:45] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 15:29:04,262 - app - get():165]ERROR in user: 'NoneType' object is not iterable
2022-06-29 15:29:04,263 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:29:04] "[31m[1mGET /confirm/%3Cclass%20'Exception'%3E HTTP/1.1[0m" 401 -
2022-06-29 15:30:37,864 - werkzeug - _log():224]INFO in _internal:  * Running on http://127.0.0.1:5000 (Press CTRL+C to quit)
2022-06-29 15:30:37,866 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:30:43,978 - app - generate_confirmation_token():12]ERROR in token: 'NoneType' object is not iterable
2022-06-29 15:30:49,179 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:30:49] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 15:31:05,946 - app - get():165]ERROR in user: 'NoneType' object is not iterable
2022-06-29 15:31:05,947 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:31:05] "[31m[1mGET /confirm/%3Cclass%20'Exception'%3E HTTP/1.1[0m" 401 -
2022-06-29 15:31:10,429 - app - get():165]ERROR in user: 'NoneType' object is not iterable
2022-06-29 15:31:10,430 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:31:10] "[31m[1mGET /confirm/%3Cclass%20'Exception'%3E HTTP/1.1[0m" 401 -
2022-06-29 15:31:43,013 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-29 15:31:43,204 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:38:40,557 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/config.py', reloading
2022-06-29 15:38:40,686 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:38:56,852 - werkzeug - _log():224]INFO in _internal:  * Running on http://127.0.0.1:5000 (Press CTRL+C to quit)
2022-06-29 15:38:56,853 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:39:07,581 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:39:07] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 15:39:20,495 - app - get():165]ERROR in user: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2022-06-29 15:39:20,497 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:39:20] "[31m[1mGET /confirm/ImtldmluYWxvdmVnYUBnbWFpbC5jb20i.YrxH5g.0n-mR1egJ3XO9NFl5mHr9lcEr1I HTTP/1.1[0m" 401 -
2022-06-29 15:40:45,726 - app - get():165]ERROR in user: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2022-06-29 15:40:45,729 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 15:40:45] "[31m[1mGET /confirm/ImtldmluYWxvdmVnYUBnbWFpbC5jb20i.YrxH5g.0n-mR1egJ3XO9NFl5mHr9lcEr1I HTTP/1.1[0m" 401 -
2022-06-29 15:41:48,399 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/config.py', reloading
2022-06-29 15:41:48,634 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 15:46:42,212 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 15:46:42,410 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 16:20:32,423 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 16:20:32,711 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 16:23:28,268 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/token.py', reloading
2022-06-29 16:23:28,497 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 16:24:28,386 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 16:24:28,718 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 16:24:42,442 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 16:24:42] "[35m[1mPOST /auth/signup HTTP/1.1[0m" 201 -
2022-06-29 16:26:13,984 - app - confirm_token():25]ERROR in token: Signature age 103 > 1 seconds
2022-06-29 16:26:13,985 - app - get():165]ERROR in user: (psycopg2.ProgrammingError) can't adapt type 'dict'
[SQL: SELECT "user".id AS user_id, "user".name AS user_name, "user".username AS user_username, "user".email AS user_email, "user".password AS user_password, "user".confirmed AS user_confirmed, "user".confirmed_on AS user_confirmed_on, "user".date_created AS user_date_created 
FROM "user" 
WHERE "user".email = %(email_1)s 
 LIMIT %(param_1)s]
[parameters: {'email_1': {'Error': 'token expired'}, 'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/14/f405)
2022-06-29 16:26:13,992 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 16:26:13] "[31m[1mGET /confirm/ImtldmluYWxvdmVnYUBnbWFpbC5jb20i.YrxSjg.UgV3le1XmVubx3r1UMJIs3RKu9U HTTP/1.1[0m" 401 -
2022-06-29 16:35:31,622 - app - confirm_token():25]ERROR in token: Signature age 661 > 1 seconds
2022-06-29 16:35:31,623 - app - get():165]ERROR in user: (psycopg2.ProgrammingError) can't adapt type 'dict'
[SQL: SELECT "user".id AS user_id, "user".name AS user_name, "user".username AS user_username, "user".email AS user_email, "user".password AS user_password, "user".confirmed AS user_confirmed, "user".confirmed_on AS user_confirmed_on, "user".date_created AS user_date_created 
FROM "user" 
WHERE "user".email = %(email_1)s 
 LIMIT %(param_1)s]
[parameters: {'email_1': {'Error': 'token expired'}, 'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/14/f405)
2022-06-29 16:35:31,624 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 16:35:31] "[31m[1mGET /confirm/ImtldmluYWxvdmVnYUBnbWFpbC5jb20i.YrxSjg.UgV3le1XmVubx3r1UMJIs3RKu9U HTTP/1.1[0m" 401 -
2022-06-29 16:45:54,332 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 16:45:54,519 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 16:48:07,829 - werkzeug - _log():224]INFO in _internal: 127.0.0.1 - - [29/Jun/2022 16:48:07] "[35m[1mPOST /auth/login HTTP/1.1[0m" 202 -
2022-06-29 16:48:49,399 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 16:48:49,776 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 16:54:03,673 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 16:54:03,887 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 16:56:41,318 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 16:56:41,496 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 16:58:20,459 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 16:58:20,665 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:00:17,503 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:00:17,691 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:04:14,303 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:04:14,580 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:05:14,620 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:05:15,067 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:06:16,030 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:06:16,263 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:07:10,740 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:07:11,104 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:08:54,044 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:08:54,299 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:12:54,116 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:12:54,412 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:13:51,773 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:13:52,015 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:17:14,615 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:17:14,902 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:18:33,884 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:18:34,121 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:20:53,006 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:20:53,182 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:22:25,722 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/app/user.py', reloading
2022-06-29 17:22:25,958 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:22:39,706 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:22:39,974 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
2022-06-29 17:23:02,234 - werkzeug - _log():224]INFO in _internal:  * Detected change in '/home/alovega/Downloads/Projects/python/StackOverflow-Lite/test/test_user_login.py', reloading
2022-06-29 17:23:02,520 - werkzeug - _log():224]INFO in _internal:  * Restarting with stat
